### PR TITLE
Wip

### PR DESCRIPTION
**Pull request by vicfriedman**
macros and ruby docs

### CODE REVIEW

**Comment by vicfriedman** [comment]
and tests!

File: README.md [hunk 1]
```diff
@@ -57,7 +57,9 @@ Run this migration using `rake db:migrate`. Awesome job!
 
 ### ActiveRecord's `has_secure_password`
 
-Next, let's update our user model so that it includes `has_secure_password`. This ActiveRecord macro gives us access to a few new methods. It works in conjunction with `bcrypt` and gives us the ability to update a password, validate the presence of a password, and authenticate a user. 
+Next, let's update our user model so that it includes `has_secure_password`. This ActiveRecord macro gives us access to a few new methods. A macro is a method that when called, creates methods for you. This is meta programming, which you don't need to worry about now. Just know that using a macros is just like calling a normal ruby method.
+
+In this case, the macro `has_secure_password` is being called just like a normal ruby method. It works in conjunction with `bcrypt` and gives us the ability to update a password, validate the presence of a password, and authenticate a user.
```
Comment (jmburges) [inline README.md]: "it works conjunction with a **gem** called bcrypt"

Comment (jmburges) [inline README.md]: and gives us all of those abilities in a secure way that doesn't actually store the plain text password in the DB

**Comment by jmburges** [comment]
super small issue and :shipit:

**Pull request merged**